recipes/iqtree/meta.yaml

```diff
@@ -16,14 +16,15 @@ requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
+    - clangdev  # [osx]
```

Comment:
Hey @andrewsanchez - maybe `host` still needs `clangdev`? Not sure why, but you could try putting this line back into the `host` section, below.

The [CI run](https://circleci.com/gh/bioconda/bioconda-recipes/98781?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) had this little tidbit of a failure: `ld: library not found for -lcrt1.10.6.o`, which is why I am wondering if we need `clangdev` in `host`.

Reply:
Good idea.  and right below that it says  `clang-4.0: error: linker command failed with exit code 1` which seems like it could be related to the last bit of info right before it exits:  `Linking CXX executable iqtree`